dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/gdv/GdvFramework.kt

```diff
@@ -0,0 +1,533 @@
+package org.dataland.frameworktoolbox.frameworks.gdv
```

Comment:
Just one general question for my undertanding: Here the special cases for the framework are defined, correct`?

Reply:
you would still need the file if no special fields would be there, but yes

Reply:
As Marcel mentioned, you need this file to make the devtools aware that there is a new framework to generate code for. It also references where the .csv file for this framework lies.

Another purpose of this file is to customize fields etc.

Reply:
thank you